AgentSessionWeb/src/shared/styles.css

```
@@ -0,0 +1,161 @@
+.log-line.stdout .log-label {
+  color: #96d894;
+}
+
+.log-line.stderr .log-label,
+.log-line.error .log-label {
+  color: #ff9a8d;
+}
+
+.log-text {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)